src/output.ejs.ftml

```diff
@@ -77,3 +77,22 @@ The following <%- category._relationships.length === 1 ? "applies" : "apply" %>
 [[/tab]]
 <% }) %>
 [[/tabview]]
+
+[!-- Tag description TOML for http://crom.wikidot.com/tag-search --]
```

Comment:
The code block will be visible in the tag documentation, so I'd either add a visible heading or hide the code block as well

Reply:
Yeah I thought this would be helpful, but other parts of the don't have it so idk.